app/controllers/onboarding_controller.rb

```diff
@@ -0,0 +1,19 @@
+class OnboardingController < ApplicationController
+  before_action :authenticate
+
+  def hood_and_houses
+    permitted_params = params.permit(entries: Hood::Onboarder::RequiredFields)
+
+    entries = permitted_params.to_h[:entries]
+
+    Hood::Onboarder.run(entries)
+  end
+
+  private
+
+  def authenticate
+    authenticate_or_request_with_http_token do |token, options|
```

Comment:
This ensures that there is a Bearer token set.

For more info, https://guides.rubyonrails.org/action_controller_overview.html#http-token-authentication

Reply:
So the client will have to send this token as part of the request header, got it 👍🏽